marketing/copy.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Marketing Copy
+description: Text to use on sites which promote Mathesar
+published: true
+date: 2023-02-23T00:00:00.000Z
+tags: 
+editor: markdown
+dateCreated: 2023-02-23T00:00:00.000Z
+---
+
+## Tagline
+
+### Technical
+
+> Mathesar is an open source web application providing an intuitive spreadsheet-like interface to PostgreSQL databases.
+
+
+### Non-technical
+
+> Mathesar is an application for managing structured data from an intuitive spreadsheet-like web interface.
+
+
+## Description
+
+### Technical
+
+> `{tagline_technical}`
+>
+> Features such as foreign keys, data types, and schemas directly reflect the behavior of Postgres, allowing users of all skill levels to collaboratively work with the same normalized relational data that other applications might use too. You can model your data, alter its structure, quickly enter data, and even generate reports. Self-hosting Mathesar gives you control, ownership, and privacy of your data.
+
+### Non-technical
+
+_(varies depending on context)_
+
+
+## Mission
+
+> Our mission is to build a tool that lets people of all skill levels use data to solve complex problems.
```

Comment:
I've got lots of thoughts on the mission, but I think it might make more sense for me to write a draft later than to try and critique this.

Reply:
I'm trying to improve this text on our [GH sponsors page](https://github.com/sponsors/centerofci)

> Our mission is to build a tool to make **understanding and working with data easy for _everyone_.**

Specifically, the text _"to make understanding and working with"_ is kind of a mouthful. I find it hard to parse smoothly.

I also wanted to tie into CCI's mission better. But I'm fine with letting go of that.

We could try:

> Our mission is to build a tool that makes understanding and working with data easy for everyone.

or

> Our mission is to build a tool that makes it easy for everyone to understand and work with data.

or

> Our mission is to make it easy for everyone to understand and work with data.

Or we could remove the mission part entirely, just starting the "Fundraising" section off with _"As a non-profit project..."_

I don't feel too strongly about this piece of content.